central/reports/common/report_formatter.go

```diff
@@ -115,7 +120,11 @@ func Format(deployedImagesResults []DeployedImagesResult, watchedImagesResults [
 							strconv.FormatFloat(v.Cvss, 'f', 2, 64),
 							discoveredTs,
 							v.Link,
-						})
+						}
+						if includeNvd == true {
+							row = append(row, strconv.FormatFloat(v.Nvdcvss, 'f', 2, 64))
```

Comment:
Can you add a func `AppendToValue(val Value, field string)` or something to `pkg/csv/writer.go` . That we we don't have to know that Value is a string slice here.